src/manifests/test_manifest.py

```diff
@@ -117,12 +130,30 @@ def __create__(self, data: dict) -> 'TestComponent':
         return TestComponent(data)
 
 
+class ClusterConfig:
+    def __init__(self, cluster_config_data: dict):
+        self.cluster_name = cluster_config_data['cluster_name']
+        self.data_nodes = cluster_config_data['data_nodes']
+        self.cluster_manager_nodes = cluster_config_data['cluster_manager_nodes']
+
+
+class TestComponentTopology:
+    def __init__(self, data: dict):
+        if data is not None:
```

Comment:
Add a class called `ClusterConfigs` that is a collection and takes `data` as an argument and does the iteration over the configs.